theano/tensor/nnet/nnet.py

```diff
@@ -2448,6 +2448,31 @@ def elu(x, alpha=1):
     return tensor.switch(x > 0, x, alpha * tensor.expm1(x))
 
 
+def selu(x):
+    """Compute the element-wise Scaled Exponential Linear unit.
+
+    .. versionadded:: 0.9.0
+
+    Parameters
+    ----------
+    x : symbolic tensor
+        Tensor to compute the activation function for.
+
+    Returns
+    -------
+    symbolic tensor
+        Element-wise scaled exponential linear activation function applied to `x`.
+
+    References
+    ----------
+    .. [1] Klambauer G, Unterthiner T, Mayr A, Hochreiter S.
```

Comment:
Seems like citation numbers are [global](http://www.sphinx-doc.org/en/stable/rest.html#citations) in Sphinx. Fixed them all here.